### PR TITLE
Add some contents to the README file

### PR DESCRIPTION
**Pull request by fredmaggiowski**
Hey, I've done some minor changes to the README to add some contents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,23 @@
 
 Head to the [Discussion Section][mia-community] to join!
 
-If you wish to request the addition of a new component to Mia-Platform marketplace you can use the [Marketplace contribution template][mia-marketplace-contribution].
-
 </div>
 
+# About this space
+
+The Mia-Platform community is an open space for everyone using our products, any question, information or feedback is welcome so feel free to join us!
```
Comment (fredmaggiowski) [inline README.md:13]:
```suggestion
The Mia-Platform community is an open space for everyone using our products; any question, information or feedback is welcome so feel free to join us!
```

File: README.md [hunk 2]
```diff
@@ -6,10 +6,23 @@
...
+
+# Mia-Platform Documentation
+
+If you to go solo you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs]
```
Comment (FilippoRezzonico) [inline README.md:17]: Manca forse questo?
```suggestion
If you wish to go solo you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs]
```

Comment (fredmaggiowski) [inline README.md:17]:
```suggestion
If you want to go solo and learn everything by yourself, you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs].
```

File: README.md [hunk 3]
```diff
@@ -6,10 +6,23 @@
...
+
+# Mia-Platform Marketplace
+
+The [Mia-Platform Markteplace][mia-marketplace] is the place where you can find everything you need to, from open source code templates to full-fledged microservices, you can customize the marketplace with your own components, if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!
```
Comment (fredmaggiowski) [inline README.md:21]:
```suggestion
The [Mia-Platform Marketplace][mia-marketplace] is the place where you can find everything you need to in order to speed up the development of your application! 

From open source code templates to full-fledged microservices, moreover you can customize the marketplace with your own components; if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!
```

File: README.md (hunk 2 above)
Comment (fredmaggiowski) [reply README.md]: I've applied a different suggestion but kept your fix, thanks

File: README.md [hunk 4]
```diff
@@ -6,10 +6,25 @@
...
+
+# Mia-Platform Documentation
+
+If you wish to go solo and learn everything by yourself, you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs].
+
+# Mia-Platform Marketplace
+
+The [Mia-Platform Marketplace][mia-marketplace] is the place where you can find everything you need to in order to speed up the development of your application! 
+
+From open source code templates to full-fledged microservices, moreover you can customize the marketplace with your own components; if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!
```
Comment (peppemanzi) [inline README.md:23]:
```suggestion
From open source code templates to full-fledged microservices, you can customize the Marketplace with your own components; if you wish to add yours feel free to [open an issue here with the Marketplace contribution template][mia-marketplace-contribution] we will be happy to help you!
```

File: README.md [hunk 5]
```diff
@@ -6,10 +6,25 @@
...
-
 </div>
 
+# About this space
+
+The Mia-Platform community is an open space for everyone using our products; any question, information or feedback is welcome so feel free to join us!
+
+# Mia-Platform Documentation
+
+If you wish to go solo and learn everything by yourself, you can find all the information you need inside the [Mia-Platform Documentation Portal][mia-docs].
```
Comment (davidebianchi) [inline README.md:17]:
```suggestion
If you wish to go solo and learn everything by yourself, you can find all the information you need inside the [Mia-Platform Documentation][mia-docs].
```

File: README.md [hunk 6]
```diff
@@ -6,10 +6,25 @@
 
 Head to the [Discussion Section][mia-community] to join!
 
-If you wish to request the addition of a new component to Mia-Platform marketplace you can use the [Marketplace contribution template][mia-marketplace-contribution].
-
 </div>
 
+# About this space
+
+The Mia-Platform community is an open space for everyone using our products; any question, information or feedback is welcome so feel free to join us!
```
Comment (davidebianchi) [inline README.md:13]: We should add a contributing file to link here, where we explain to not insert clients name or information and other reserved information (e.g. some secret values)

Comment (fredmaggiowski) [reply README.md:13]: That's a good insight, do you have time to write it?

I could bootstrap it but I guess the contributing rules can be a lot @akelity you might want to participate in it as well?

File: README.md [hunk 7]
```diff
@@ -6,10 +6,25 @@
...
+
+# Mia-Platform Documentation
+
+If you wish to go solo and learn everything by yourself, you can find all the information you need inside [Mia-Platform Documentation Portal][mia-docs].
```
Comment (fredmaggiowski) [inline README.md:17]:
```suggestion
If you wish to go solo and learn everything by yourself, you can find all the information you need inside [Mia-Platform Documentation][mia-docs].
```

File: README.md (hunk 6 above)
Comment (fredmaggiowski) [reply README.md:13]: I've added the file with a basic structure of paragraph I think may be useful; I did not write them all yet, feel free to add yours

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+## Code of Conduct
+
+This project adheres to the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md), please read it and follow it
+before contributing. If you find someone that is not respecting it please report its behaviour.
+
+## How Can I Contribute
+
+### Discussion Board
+
```
Comment (fredmaggiowski) [inline CONTRIBUTING.md:15]:
```suggestion

The [discussion board](https://github.com/mia-platform/community/discussions) is the place where you can search for common problems/ideas and ask your own; each discussion should be properly labelled, formatted and written following the guidelines below:

- all threads must be written in English
- each thread *must not* contain any reserved information (customer names, identifiers, personal data, secret values, etc)
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+#### Create a new discussion
+
```
Comment (fredmaggiowski) [inline CONTRIBUTING.md:17]:
```suggestion

Before creating a new discussion make sure there is no discussion on the same topic, already; if you didn't found what you are looking for, open a new one and make sure to properly label it.
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+before contributing. If you find someone that is not respecting it please report its behaviour.
+
+## How Can I Contribute
+
+### Discussion Board
+
+The [discussion board](https://github.com/mia-platform/community/discussions) is the place where you can search for common problems/ideas and ask your own; each discussion should be properly labelled, formatted and written following the guidelines below:
+
+- all threads must be written in English
+- each thread *must not* contain any reserved information (customer names, identifiers, personal data, secret values, etc)
```
Comment (fredmaggiowski) [inline CONTRIBUTING.md:19]:
```suggestion
- each thread *must not* contain any reserved information (customer names, identifiers, personal data, secret values, etc)

```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+#### Create a new discussion
+
+Before creating a new discussion make sure there is no discussion on the same topic, already; if you didn't found what you are looking for, open a new one and make sure to properly label it.
```
Comment (fredmaggiowski) [inline CONTRIBUTING.md:22]:
```suggestion
Before creating a new discussion make sure there is no discussion on the same topic, already; if you didn't found what you are looking for, open a new one and make sure to properly label it.

```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
+# Contribution Guidelines
+
+First off, thank you for considering contributing to Mia-Platform Community.
+
+This document is intended to provide guidelines for contributing to the community both with contents and with the discussion board.
```
Comment (ugho16) [inline CONTRIBUTING.md:5]:
```suggestion
This document aims to provide guidelines for contributing to the community, both with contents and with the discussion board.
```

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+- each thread *must not* contain any reserved information (customer names, identifiers, personal data, secret values, etc)
+- proper labels must be used, there's a bunch of labels already available, if you don't find a suitable one we can help
+
+#### Create a new discussion
+
+Before creating a new discussion make sure there is no discussion on the same topic, already; if you don't found what you are looking for, open a new one and make sure to properly label it.
+
+### Change Community contents
+
+To propose changes to the community contents you can open a new pull request if you are not sure where to start from you can open an issue to discuss your proposal.
```
Comment (ugho16) [inline CONTRIBUTING.md:28]:
```suggestion
To propose changes to the community contents, you can open a new pull request. If you are not sure where to start from, you can open an issue to discuss your proposal.
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+
+The [discussion board](https://github.com/mia-platform/community/discussions) is the place where you can search for common problems, ideas and ask your own; each discussion (regardless if it is a question or an answer) must follow the guidelines below:
+
+- all threads must be written in English
+- each thread *must not* contain any reserved information (customer names, identifiers, personal data, secret values, etc)
+- proper labels must be used, there's a bunch of labels already available, if you don't find a suitable one we can help
+
+#### Create a new discussion
+
+Before creating a new discussion make sure there is no discussion on the same topic, already; if you don't found what you are looking for, open a new one and make sure to properly label it.
```
Comment (ugho16) [inline CONTRIBUTING.md:24]:
```suggestion
Before creating a new discussion make sure there is no discussion on the same topic, already; if you did not find what you were looking for, open a new one and make sure to properly label it.
```

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+This project adheres to the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md), please read it and follow it
+before contributing. If you find someone that is not respecting it please report its behaviour.
+
+## How Can I Contribute
+
+### Discussion Board
+
+The [discussion board](https://github.com/mia-platform/community/discussions) is the place where you can search for common problems, ideas and ask your own; each discussion (regardless if it is a question or an answer) must follow the guidelines below:
+
+- all threads must be written in English
```
Comment (ugho16) [inline CONTRIBUTING.md:18]:
```suggestion
- All threads must be written in English;
```

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+- each thread *must not* contain any reserved information (customer names, identifiers, personal data, secret values, etc)
```
Comment (ugho16) [inline CONTRIBUTING.md:19]:
```suggestion
- Each thread *must not* contain any reserved information (customer names, identifiers, personal data, secret values, etc.);
```

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+- proper labels must be used, there's a bunch of labels already available, if you don't find a suitable one we can help
```
Comment (ugho16) [inline CONTRIBUTING.md:20]:
```suggestion
- Proper labels must be used; there's a bunch of labels already available, if you don't find a suitable one, ask the maintainers.
```

**Pull request merged**